packages/react/src/alertbox/Alertbox.tsx

```diff
@@ -0,0 +1,200 @@
+   * @default true
+   */
+  isDismissable?: boolean;
+  /** Additional CSS className for the element. */
+  className?: string;
+  /**
+   * Controls if the alert is rendered or not.
+   * This is used to control the open/closed state of the component; make the component "controlled".
+   */
+  isVisible?: boolean;
+  /**
+   * Callback that should be triggered when a dismissable alert is closed.
+   * This is used to control the open/closed state of the component; make the component "controlled".
+   */
+  onClose?: () => void;
+};
+
+const Alertbox = ({
+  children,
+  role,
+  className,
+  variant = 'info',
+  isDismissable,
+  isVisible: isControlledVisible,
+  onClose,
+}: Props) => {
+  const Icon = iconMap[variant];
+
+  const [isUncontrolledVisible, setIsUncontrolledVisible] = useState(true);
+
+  const close = () => {
+    setIsUncontrolledVisible(false);
+    if (onClose) onClose();
+  };
+
+  if (onClose && !isDismissable) {
+    console.warn(
+      'Passing an `onClose` callback without setting the `isDismissable` prop to `true` will not have any effect.',
+    );
+  }
+
+  const isVisible =
+    isControlledVisible !== undefined
+      ? isControlledVisible
+      : isUncontrolledVisible;
+
+  // Set a default aria-label for the close button and handle translations based on the current locale
+  const { locale } = useLocale();
+  let closeLabel = 'Lukk';
+  if (locale === 'sv') closeLabel = 'Stäng';
+  else if (locale === 'en') closeLabel = 'Close';
+
+  if (!children) {
+    console.error('`No children was passed to the <AlertBox/>` component.');
+    return;
+  }
+
+  const [firstChild, ...restChildren] = Children.toArray(children);
+
+  return (
+    isVisible && (
+      <div
+        className={alertVariants({
+          className,
+          variant,
+        })}
+        role={role}
+      >
+        <Icon />
+        {firstChild}
+        {isDismissable && (
+          <Button
+            className="-m-2 grid h-11 w-11 place-items-center outline-transparent transition-[outline] duration-200 focus:-outline-offset-8 focus:outline-black"
+            onPress={close}
+            aria-label={closeLabel}
+          >
+            <Close />
+          </Button>
+        )}
+        {restChildren}
+      </div>
+    )
+  );
+};
+
+type AlertboxHeadingProps = {
+  children: React.ReactNode;
+  /** The level of the heading */
+  level: 1 | 2 | 3 | 4 | 5 | 6;
+  /** Additional CSS className for the element. */
+  className?: string;
+};
+
+const AlertboxHeading = ({
+  children,
+  level,
+  className,
+}: AlertboxHeadingProps) => {
+  const Heading = `h${level}` as const;
+  return (
+    <Heading className={cx(className, 'text-base font-medium leading-7')}>
+      {children}
+    </Heading>
+  );
+};
+
+type AlertboxBodyProps = {
+  children: React.ReactNode;
+  /** Additional CSS className for the element. */
+  className?: string;
+};
+
+const AlertboxBody = ({ children, className }: AlertboxBodyProps) => (
```

Comment:
Forslag: Kanskje vi skal kalle denne AlertContent i stedet?

Om vi får det til, i fremtiden, hadde det vært kult med generiske komponenter som Heading, Footer, Content som kan brukes i feks Card og Alert, uten at vi nødvendigvis har egen CardFooter og AlertFooter. Se [Spectrum for eksempel](https://react-spectrum.adobe.com/react-spectrum/InlineAlert.html).

Og hvis vi får til noe sånt, da føler jeg at `<Content>` er et bedre navn enn `<Body>`

Reply:
Jeg er enig i at en generisk `<Content/>` er bedre enn `<Body/>`. Vi kan godt endre til AlertContent her